assets/scss/media.scss

```diff
@@ -0,0 +1,34 @@
+.swiper {
+  .iiif-frame-wrapper {
+    height: 100%;
+    width: 1140px;
+    max-width: 100vw;
+    padding-right: 20px;
+    padding-left: 20px;
+
+    @media (min-width: $bp-small) {
+      max-width: 540px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    @media (min-width: $bp-medium) {
+      max-width: 720px;
+    }
+
+    @media (min-width: $bp-large) {
+      max-width: 960px;
+    }
+
+    @media (min-width: $bp-extralarge) {
+      max-width: 1140px;
+    }
```

Comment:
Okay, a thought: if there is just one slide, the slide can have a width of 100%. (When smaller, the media will be centred). 

In that case, I think you don't need the iiif-frame-wrapper class. You could make use of the existing "container" class. That is a Bootstrap class and handles the width exactly as you have described here.  And you could use the "h-100" class to set the height.

Reply:
We don't want the viewer to be so wide. It leads to strange aspect ratios. Mainly because of the max height of 568px on the whole swiper.

Reply:
I misunderstood your comment initially. I had another look though and yes, that works and is simpler.